test/infra.tf

```diff
@@ -0,0 +1,93 @@
+provider "aws" {
```

Comment:
We have an almost identical code in the `hack/prebuild` dir. I think this PR should either utilize it or delete it, to prevent drift.

Reply:
It would suggest to copy all we can from the `infra.tf` into `hack/prebuild`. The `infra.tf` contains more rules, role and instance profile creation, and other finds I found on the way to successfully bootstrap the k8s cluster with the `aws-actuator` binary.